src/pages/NewConversation/GroupInfoForm/GroupInfoForm.js

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import { Redirect } from 'react-router-dom';
+
+// Components
+import Spinner from '../../../layout/Spinner/Spinner';
+import BackLink from '../../../components/BackLink/BackLink';
+
+// Hooks
+import useGroupInfoForm from './useGroupInfoFrom';
+import { ReactComponent as CreateSVG } from '../../../assets/usersadded.svg';
+
+// Styles
+import classes from './GroupInfoForm.module.scss';
+import {
+  CONVERSATION_WITHOUT_ID,
+  NEW_CONVERSATION_GROUP_RECEIVERS,
+} from '../../../consts/routes';
+
+function GroupInfoForm() {
+  const {
+    chatName,
+    handleChatName,
+    handleGroupAvatarChange,
+    previewGroupAvatar,
+    handleCreateConversation,
+    receivers,
+    isCreating,
+    createdConversation,
+  } = useGroupInfoForm();
+
+  if (createdConversation) {
+    return (
+      <Redirect
```

Comment:
This is a bizarre way of doing redirects.  You should opt out from putting such a business logic into jsx